libmcount/dynamic.c

```diff
@@ -242,9 +249,36 @@ static int do_dynamic_update(struct symtabs *symtabs, char *patch_funcs,
 	strv_for_each(&funcs, name, j) {
 		bool found = false;
 		bool csu_skip;
+		char *modname, *mod_delimiter;
 		unsigned i, k;
 		struct sym *sym;
 		struct uftrace_pattern patt;
+		struct uftrace_mmap *map;
+		struct mcount_dynamic_info *mdi = NULL;
+
+		mod_delimiter = strchr(name, '@');
```

Comment:
I don't like long names in general.  How about just call it `mod` (or just `m`)?

Reply:
changed to delim, is it ok?

Reply:
yep